### PR TITLE
Feature/us16 forgot my password

### PR DESCRIPTION
**Pull request by jpc11623**

### CODE REVIEW

File: Gateway/crds-angular/Controllers/API/LoginController.cs [hunk 1]
```diff
@@ -18,19 +15,53 @@ namespace crds_angular.Controllers.API
...
+                return this.InternalServerError();
+            }
+        }
+
+        [HttpPost]
+        [Route("api/resetpasswordverify/")]
+        public IHttpActionResult AcceptResetRequest(PasswordResetVerification request)
+        {
+            // Worked in successor story
+            throw new NotImplementedException();
```
Comment (IngageBrewerM) [inline Gateway/crds-angular/Controllers/API/LoginController.cs]: Should this just be removed until the successor story is worked?

File: CI/SQL/US16-01-AddPasswordResetTokenToUsers.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+USE [MinistryPlatform]
+GO
+
+IF NOT EXISTS(
+	SELECT * FROM sys.columns
+	WHERE Name = N'__PasswordResetToken' and OBJECT_ID = OBJECT_ID(N'dp_Users'))
+BEGIN
+
+ALTER TABLE dp_Users
+ADD __PasswordResetToken VARCHAR(MAX) NULL
```
Comment (IngageBrewerM) [inline CI/SQL/US16-01-AddPasswordResetTokenToUsers.sql]: Weren't the underscores removed from this field?

File: Gateway/crds-angular/Services/PersonService.cs [hunk 3]
```diff
@@ -65,5 +65,12 @@ public Person GetLoggedInUserProfile(String token)
 
             return person;
         }
+
+        public bool ResetPassword(string email)
+        {
+
+
+            return true;
+        }
```
Comment (IngageBrewerM) [inline Gateway/crds-angular/Services/PersonService.cs]: What does this do?

File: Gateway/crds-angular/Controllers/API/LoginController.cs [hunk 4]
```diff
@@ -18,19 +15,53 @@ namespace crds_angular.Controllers.API
...
+            //try
+            //{
+            //    _loginService.AcceptPasswordResetRequest(request.Email, request.Token, request.Password);
+            //    return this.Ok();
+            //}
+            //catch (Exception)
+            //{
+            //    return this.InternalServerError();
+            //}
+        }
```
Comment (IngageBrewerM) [inline Gateway/crds-angular/Controllers/API/LoginController.cs]: At least these comments? :)

File: Gateway/crds-angular/Services/PersonService.cs (hunk 3 above)
Comment (jpc11623) [inline Gateway/crds-angular/Services/PersonService.cs]: Removed

File: Gateway/crds-angular/Controllers/API/LoginController.cs (hunk 4 above)
Comment (jpc11623) [inline Gateway/crds-angular/Controllers/API/LoginController.cs]: Only those comments.

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**